### DISCUSSION TITLE
How to use the ASE calculator without python in deepmd directory?

### DISCUSSION THREAD

**Discussion by phys-chem** [deepmodeling/deepmd-kit#2911] [General]
I want to use other python verison to call DP calculator. I copy calculator.py but it does not work. Is there a python package called "deepmd" importable?

**Comment by wanghan-iapcm** [comment]
If you have successfully installed deepmd-kit, you should be able to import `deepmd`
```
python -c "import deepmd"
```

**Reply by phys-chem** [reply]
Thank you for reply! It works when using this directory "deepmd-kit/bin/python". However, I want to use python in other directories. What should I do？Thank you very much!

**Reply by wanghan-iapcm** [reply]
it seems that you haven't successfully installed the deepmd-kit. https://docs.deepmodeling.com/projects/deepmd/en/v2.2.7/getting-started/install.html#easy-install

**Comment by phys-chem** [comment]
> it seems that you haven't successfully installed the deepmd-kit. https://docs.deepmodeling.com/projects/deepmd/en/v2.2.7/getting-started/install.html#easy-install

Thank you for kindly help! I can use the "dp" command for training and can import "deepmd" when using "deepmd-kit/bin/python". I want to use "local/python" to call  deepmd-ASE calculator. I follow this link for installing. Is there a problem with the installation of deepmd?

**Comment by phys-chem** [comment]
I am sorry for misunderstanding. It has been solved by install the python interface of deepmd.
